.travis.yml

```diff
@@ -1,36 +1,16 @@
-# This config file for Travis CI utilizes ros-industrial/industrial_ci package.
-# For more info for the package, see https://github.com/ros-industrial/industrial_ci/blob/master/README.rst
-sudo: required
-dist: trusty
 services:
   - docker
-language: generic
-compiler:
-  - gcc
-notifications:
-  email:
-    on_success: always
-    on_failure: always
-    recipients:
-      - tony.baltovski@gmail.com
-env:
-  matrix:
-    - ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - ROS_DISTRO="indigo" ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
-    - ROS_DISTRO="indigo" PRERELEASE=true
-    - ROS_DISTRO="kinetic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - ROS_DISTRO="kinetic"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
-    - ROS_DISTRO="kinetic"   PRERELEASE=true
-    - ROS_DISTRO="lunar"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros/ubuntu
-    - ROS_DISTRO="lunar"   ROS_REPOSITORY_PATH=http://packages.ros.org/ros-shadow-fixed/ubuntu
-    - ROS_DISTRO="lunar"   PRERELEASE=true
-matrix:
-  allow_failures:
-    - env: ROS_DISTRO="indigo"  PRERELEASE=true  # Run docker-based ROS prerelease test http://wiki.ros.org/bloom/Tutorials/PrereleaseTest Because we might not want to run prerelease test for all PRs, it's omitted from pass-fail criteria.
-    - env: ROS_DISTRO="kinetic" PRERELEASE=true
-    - env: ROS_DISTRO="lunar"   PRERELEASE=true
+
 install:
-  - git clone https://github.com/ros-industrial/industrial_ci.git .ci_config
-script:
-  - source .ci_config/travis.sh
-#  - source ./travis.sh  # Enable this when you have a package-local script
+  - git clone --quiet --depth 1 https://github.com/mikaelarguedas/ros2ci.git .ros2ci
+
+matrix:
+  include:
+    - env: JOB_TYPE=dashing
+      script: .ros2ci/travis.bash $JOB_TYPE
+    - env: JOB_TYPE=eloquent
+      script: .ros2ci/travis.bash $JOB_TYPE
+    - env: JOB_TYPE=nightly
+      script:
+      - cp additional_repos.repos .ros2ci/
```

Comment:
For all jobs to build these packages from source, this should be put in the `install` step. Currently only the `nightly` job builds the dependencies from source explaining why the other jobs fail to resolve `urg_c` rosdep key.

Reply:
Let's see if that works with https://github.com/ros-drivers/urg_node/pull/50/commits/1882fe6ce8ae9db24b9e6e534656f533939694d2

thanks for the quick response!